### ISSUE TITLE
String encoding issues when using formatter

### ISSUE THREAD

**Issue by DriesOeyen** [sdispater/pendulum#168]
Hi all! We're building a web application in which we use the alternative formatter on Pendulum objects right in a bunch of Jinja2 templates. Think:

```
<li>Account creation date: {{ user.creation_datetime.format('LLL') }}</li>
```

This is working great most of the time, but crashes and burns spectacularly in some cases. We're using Python 2.7, and it appears that the Pendulum formatters return regular (ASCII) strings. However, those strings contain Unicode characters in some instances. We discovered this problem when attempting to format a datetime in French. Pendulum would return the "14 décembre 2017" as `'14 d\xc3\xa9cembre 2017'` rather than `u'14 d\xe9cembre 2017'`. In other words, it's improperly stuffing the Unicode character "é" in an ASCII string. Attempting to cast this invalid ASCII string as a Unicode string will bomb:

```
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 4: ordinal not in range(128)
```

Of course, casting ASCII strings as Unicode is exactly what Jinja2 does, so this exception crashes the entire page. In our opinion, this is a bug in Pendulum: since clearly the formatter will sometimes return Unicode characters, it should simply return Unicode strings in the first place.

We have fixed this on our end by changing line 234 of `pendulum/formatting/alternative_formatter.py` to:

```python
return trans.decode('utf-8')
```

We wouldn't mind pushing this back upstream, but we aren't intimately familiar with the codebase and need some guidance:

- Is this issue and the suggested fix in line with the opinion of the maintainers?
- Would there be any other place within the library (besides the basic and alternative formatters) where returning ASCII strings could be problematic? We're thinking "yes", because using methods like `dt.to_cookie_string()` can expose this problem as well. Some guidance regarding where this fix should be applied would be appreciated.
- Could this proposed fix have any unintended side-effects that we've overlooked?

**Comment by sdispater** [comment]
Thanks for the detailed explanation.

This is definitely an issue with Pendulum. This should be an easy fix, like you mentioned, since we just have to return a Unicode string from both formatters.

Methods like `to_cookie_string()` use `format()` under the hood so we should be good.

**Comment by sdispater** [comment]
This is fixed as of version `1.4.0`
